### PR TITLE
feature: provide external event from WebSocket

### PR DESCRIPTION
**Pull request by ka2jun8**
* based on #141 
* provide external event from WebSocket
* Response type and result(success/error)
* Large character of response key 
* Timestamp using Date.getTIme

### CODE REVIEW

File: src/main/java/io/personium/core/ws/WebSocketService.java [hunk 1]
```diff
@@ -223,110 +218,239 @@ public void onMessage(String text, Session session) {
...
+            // TODO It may be better cellRsCmp is managed by HashMap
+            Cell cell = ModelFactory.cell(cellId, null);
+            AccessContext ac = createAccessContext(accessToken, cell.getName());
+            CellCmp cellCmp = ModelFactory.cellCmp(cell);
+            if (cellCmp.exists()) {
+                CellRsCmp cellRsCmp = new CellRsCmp(cellCmp, cell, ac);
+                cellRsCmp.checkAccessContext(ac, CellPrivilege.EVENT);
+                PersoniumEvent pEvent = new PersoniumEvent(
+                        PersoniumEvent.EXTERNAL_EVENT,
+                        (String) event.get("Type"),
```
Comment (shimono) [inline src/main/java/io/personium/core/ws/WebSocketService.java]: -- Why not turn these into constants ? --
Type
Object
Info
RequestKey
Schema
Subject
Timestamp
External

Comment (shimono) [reply src/main/java/io/personium/core/ws/WebSocketService.java]: Probably these constants should not rather be defined in this class.
Classes such as Event or EventBus probably have these constants already so 
we should use them instead.

**Comment by SawamiWataru** [comment]
Shimono's change request issues PullRequest again after merging.

**Comment by shimono** [comment]
Thank you !

**Pull request merged**